sources/lib/reasoners/arith.mli

```diff
@@ -26,6 +26,14 @@
 (*                                                                            *)
 (******************************************************************************)
 
+(** Compute x^y. Raise Exit if y is not an Int (castable in Int).
+    Raise Exit if the return type is Int but the result of x^y is not.*)
+val calc_power : Numbers.Q.t -> Numbers.Q.t -> Ty.t -> Numbers.Q.t
```

Comment:
You use `x` and `y` in the comment but do not specify to which arguments they correspond (it can be guessed, but it's better for doc comments to not be ambiguous like that).